src/Microsoft.AspNetCore.Sockets/Transports/ServerSentEventsTransport.cs

```diff
@@ -37,6 +37,9 @@ public ServerSentEventsTransport(ReadableChannel<Message> application, ILoggerFa
 
             context.Response.Headers["Content-Encoding"] = "identity";
 
+            // Workaround for a Firefox bug where EventSource won't fire the open event
+            // until it receives some data
+            await context.Response.WriteAsync(":\r\n");
```

Comment:
No. Parsers will try to parse it and will barf so we will have to update them everywhere. We could send data:T\r\ndata:\r\n\r\n but it is more data and I think it will be reported as a new message so we will have to handle this in each client. What I am sending now is a valid event-source comment as per spec so it is not reported in browsers and I updated our parser in C# to ignore comments in general.